### PR TITLE
Allow osbuild-composer to run without Weldr API

### PR DESCRIPTION
**Pull request by ondrejbudai**
I think we should be able to ditch the local worker altogether. This is the first step. See the individual comments for details.

### CODE REVIEW

File: cmd/osbuild-composer/composer.go [hunk 1]
```diff
@@ -200,7 +199,17 @@ func (c *Composer) Start() error {
...
+		go func() {
+			err := c.weldr.Serve(c.weldrListener)
+			if err != nil {
+				panic(err)
+			}
+		}()
+	}
+
+	// wait indefinitely
+	select {}
```
Comment (larskarlitski) [inline cmd/osbuild-composer/composer.go:221]: Note: we probably want to stop when any of the APIs `panic`, which is easily solvable by passing a channel to each of the above goroutines. Out of scope for this PR.

**Review by larskarlitski — APPROVED** [review]
Nice! I really like this. Thanks!

File: cmd/osbuild-composer/composer.go (hunk 1 above)
Comment (ondrejbudai) [reply cmd/osbuild-composer/composer.go:221]: I think that's the default Go behaviour: https://play.golang.org/p/pL4FVhEZ3-c

**Review by teg — APPROVED** [review]
Great stuff! It is a bit fiddly to get both the RPM and systemd behavior backwards compatible, but this seems correct. Thanks!

**Pull request merged**